### ISSUE TITLE
Get notification when your circuit is forked

### ISSUE THREAD

**Issue by ShreyaPrasad1209** [CircuitVerse/CircuitVerse#574]
**Is your feature request related to a problem? Please describe.**
As of now, if we are forking someone else's circuit, they don't get a notification. So they don't get an idea of how many people are using their circuits.

**Describe the solution you'd like**
Send notification to user whenever their circuit is forked. It creates a more transparent environment. Also, we can show how many time a circuit has been forked near number of views feature.

**Are you working on this? (Yes/No)**
GCI students under difficult

**Comment by ankitkataria** [comment]
Looks approachable after #243

**Comment by junron** [comment]
Hi, I'm interested in taking this task. However, I understand that the notification system is not completed yet. Perhaps I could use email instead of a notification system until the notification system is ready? The number of forks feature shouldn't be too hard though.

Comments @ShreyaPrasad1209 @ankitkataria ?

**Comment by ankitkataria** [comment]
Go ahead! :rocket:

**Comment by junron** [comment]
I'll take another task first and do more research on #243, and maybe do a PR to that, then come back to this task! Thanks for confirming!
